modules/development/variables.tf

```
@@ -15,5 +15,5 @@ variable "machine_type" {
 }
 
 variable "k8s_version" {
-  default = "1.13.6-gke.13"
+  default = "1.13.11-gke.14"
```

I guess this will imply a redeployment of the whole cluster, I think we can wait until the release of polkadot for updating, WDYT?
```suggestion
  default = "1.13.6-gke.13"
```

---

yep. housekeeping after release.